app/page.tsx

```diff
@@ -1,10 +1,120 @@
-import { Grid } from '@mui/material'
-import React from 'react'
+import * as React from "react";
+import Grid from "@mui/material/Grid";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import IconButton, { IconButtonProps } from "@mui/material/IconButton";
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import Box from "@mui/material/Box";
+import Image from "next/image";
+import { primaryAction } from "ol/events/condition";
 
-const Index = () => {
+
```

Comment:
Why have you kept a large space like this? Please format the code before creating a PR